### PR TITLE
Register MEME pool.

### PR DESCRIPTION
**Pull request by tensafefrogs**
My editor removed some extra whitespace as well, I can split that into a separate commit if you like.

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1727,5 +1727,20 @@
...
+      "poolId":"c9aa8e864d5cc292f0e93669a79b41d67f8ea62481a0d85b14bea386",
+      "social":{
+         "facebook":"",
+         "twitter":"https://twitter.com/meme_pool",
+         "website":"https://twoangrycamelsinacar.com/",
+         "reddit":"https://www.reddit.com/user/MEME-Pool",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"sporked",
```
Comment (lophre) [inline registry.json:1742]: Can't find this username on Telegram.  We need a valid username for SPA voting.

**Review by lophre — CHANGES_REQUESTED** [review]
Please change telegram entry to a valid username

File: registry.json (hunk 1 above)
Comment (tensafefrogs) [reply registry.json]: Whoops, typo. Fixed and merged in the latest changes.

**Review by lophre — APPROVED** [review]

**Comment by lophre** [comment]
Welcome in our Alliance!

**Review by lophre — APPROVED** [review]

**Pull request merged**